iac-extensions/arm/src/main/java/org/sonar/iac/arm/tree/api/OutputDeclaration.java

```diff
@@ -24,17 +24,17 @@
 public interface OutputDeclaration extends Statement {
   Identifier name();
 
-  Expression type();
+  String type();
```

Comment:
I know this is part of the spec, and now I wonder if returning something like StringLiteral and NumericLiteral would make more sense. That way we would keep access to the range of the elements.

Reply:
Was thinking the same, It will probably make more sense for this reason.
It also seems more evident now that we have those literal classes, I will make the modification.